tests/e2e/Vagrantfile

```diff
@@ -35,8 +35,8 @@ Vagrant.configure("2") do |config|
     end
   end
 
-  config.vm.define "tests-e2e-ubuntu2004", autostart: false do |ubuntu|
-    ubuntu.vm.box = "generic/ubuntu2004"
+  config.vm.define "tests-e2e-ubuntu2204", autostart: false do |ubuntu|
```

Comment:
Hi @ldoktor !

Actually in [CI we are using both versions](https://github.com/confidential-containers/operator/blob/main/.github/workflows/ccruntime_e2e.yaml#L25). With Ubuntu 20.04 we test the code that replaces containerd in the host (related to https://github.com/confidential-containers/operator/issues/288).

So maybe create a new entry for 22.04 instead?

Reply:
I'm not really a Vagrant user and only wanted to keep things working. I can remove this commit if 2004 is still covered.

Reply:
hi @ldoktor , could you please add a new entry for 22.04 but leave the old one (20.04)?

I got a new workstation and I can give it a try with vagrant. The old laptop I had vagrant stop working after an update... :(

Reply:
I did that and while testing it the 22.04 worked well, the 20.04 failed with:

```
    tests-e2e-ubuntu2004:   export KUBECONFIG=/etc/kubernetes/admin.conf
    tests-e2e-ubuntu2004: 
    tests-e2e-ubuntu2004: You should now deploy a pod network to the cluster.
    tests-e2e-ubuntu2004: Run "kubectl apply -f [podnetwork].yaml" with one of the options listed at:
    tests-e2e-ubuntu2004:   https://kubernetes.io/docs/concepts/cluster-administration/addons/
    tests-e2e-ubuntu2004: 
    tests-e2e-ubuntu2004: Then you can join any number of worker nodes by running the following on each as root:
    tests-e2e-ubuntu2004: 
    tests-e2e-ubuntu2004: kubeadm join 192.168.122.246:6443 --token gldusw.fa2z20xf9j1n9ljm \
    tests-e2e-ubuntu2004:       --discovery-token-ca-cert-hash sha256:18a47d92ef101b8474ca6a868048f618a5b50fa1789f0ad148f6f630eab977cb
    tests-e2e-ubuntu2004: namespace/kube-flannel created
    tests-e2e-ubuntu2004: clusterrole.rbac.authorization.k8s.io/flannel created
    tests-e2e-ubuntu2004: clusterrolebinding.rbac.authorization.k8s.io/flannel created
    tests-e2e-ubuntu2004: serviceaccount/flannel created
    tests-e2e-ubuntu2004: configmap/kube-flannel-cfg created
    tests-e2e-ubuntu2004: daemonset.apps/kube-flannel-ds created
    tests-e2e-ubuntu2004: node/ubuntu2004.localdomain untainted
    tests-e2e-ubuntu2004: node/ubuntu2004.localdomain untainted
    tests-e2e-ubuntu2004: node/ubuntu2004.localdomain labeled
    tests-e2e-ubuntu2004: ::info:: Build and install the operator
    tests-e2e-ubuntu2004: ::debug:: Add repo /home/vagrant/src/confidential-containers/operator to git's safe.directory
    tests-e2e-ubuntu2004: test -s /home/vagrant/src/confidential-containers/operator/bin/controller-gen || GOBIN=/home/vagrant/src/confidential-containers/operator/bin go install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.2
    tests-e2e-ubuntu2004: /home/vagrant/src/confidential-containers/operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
    tests-e2e-ubuntu2004: /home/vagrant/src/confidential-containers/operator/bin/controller-gen: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.32' not found (required by /home/vagrant/src/confidential-containers/operator/bin/controller-gen)
    tests-e2e-ubuntu2004: /home/vagrant/src/confidential-containers/operator/bin/controller-gen: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by /home/vagrant/src/confidential-containers/operator/bin/controller-gen)
    tests-e2e-ubuntu2004: make: *** [Makefile:109: manifests] Error 1
    tests-e2e-ubuntu2004: 
    tests-e2e-ubuntu2004: ::error:: Testing failed with 2
==> tests-e2e-ubuntu2004: Attempting graceful shutdown of VM...
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

I'll look into that tomorrow... (unless you already know what is wrong)

Reply:
It looks like my fault. I had it compiled on my machine, which got transferred to the vagrant machine and failed. Interesting finding, operator doesn't come with "make clean" :-/ (I'll consider adding it). Anyway I'll finish my testing and update this PR tomorrow.